spec/features/edit_note_conflict_spec.rb

```diff
@@ -21,6 +21,9 @@
         # ...in the meantime there's an edit
         note.update_attributes! content: '<p>note content - update 1</p>'
 
+        # FIXME: clear text first to work around bug in Capybara/Poltergeist
+        find('.ql-editor').set('')
```

Comment:
@koffeinfrei without this I ended up with the text being reversed and newlines inserted:

```
       expected: "<p>note content - update 2</p>"
            got: "<p>2</p><p> </p><p>e</p><p>t</p><p>a</p><p>d</p><p>p</p><p>u</p><p> </p><p>-</p><p> </p><p>t</p>
<p>n</p><p>e</p><p>t</p><p>n</p><p>o</p><p>c</p><p> </p><p>e</p><p>t</p><p>o</p><p>n</p><p><br></p>"
```

Couldn't find out why it happens, but if I clear the text first it works fine :thinking: 

Alternatively I can downgrade Capybara again to 2.x which would probably "solve" this as well.